### PR TITLE
Add language identifiers

### PR DESCRIPTION
**Pull request by Youssef1313**
- Added some missing language identifiers.
- Fixed some bad indentations.
- Fixed some code that had `Throw` instead of `throw` and `Else` instead of `else` which will cause compile error in C#.

Related to #2192 

edit by @mairaw: added the related issue

### CODE REVIEW

File: docs/framework/wcf/extending/how-to-import-custom-wsdl.md [hunk 1]
```diff
@@ -67,8 +67,9 @@ This topic describes how to import custom WSDL. To handle the custom WSDL, you m
   
 3. Create a new <xref:System.ServiceModel.Description.WsdlImporter> instance (passing in the <xref:System.ServiceModel.Description.MetadataSet> instance that contains the WSDL documents that you want to import), and call <xref:System.ServiceModel.Description.WsdlImporter.ImportAllContracts%2A>:  
   
-    ```  
-    WsdlImporter importer = new WsdlImporter(metaDocs);          System.Collections.ObjectModel.Collection<ContractDescription> contracts  = importer.ImportAllContracts();  
+    ```csharp
+    WsdlImporter importer = new WsdlImporter(metaDocs);
+	System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();
```
Comment (BillWagner) [inline docs/framework/wcf/extending/how-to-import-custom-wsdl.md]: The tab character is causing extra indentation on the second line.  I'd suggest the following:

```suggestion
    System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();  
```

File: docs/framework/wcf/extending/how-to-write-an-extension-for-the-servicecontractgenerator.md [hunk 2]
```diff
@@ -66,21 +66,22 @@ This topic describes how to write an extension for the <xref:System.ServiceModel
...
     MetadataSet metaDocs = mexClient.GetMetadata();  
     ```  
   
 5. Create a `WsdlImporter` and call `ImportAllContracts`.  
   
-    ```  
-    WsdlImporter importer = new WsdlImporter(metaDocs);            System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();  
+    ```csharp
+    WsdlImporter importer = new WsdlImporter(metaDocs);
+	System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();
```
Comment (BillWagner) [inline docs/framework/wcf/extending/how-to-write-an-extension-for-the-servicecontractgenerator.md]: Same tab to space conversion:

```suggestion
    System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();  
```

File: docs/framework/wcf/samples/custom-wsdl-publication.md [hunk 3]
```diff
@@ -65,43 +65,46 @@ public interface ICalculator
...
+            // Set the doc element; if this isn't done first, there is no XmlElement in the
+            // DocumentElement property.
+            operation.Documentation = String.Empty;
+
+            // Operation C# triple comments.
+            XmlDocument owner = operation.DocumentationElement.OwnerDocument;
+            XmlElement newSummaryElement = owner.CreateElement("summary");
+            newSummaryElement.InnerText = this.Text;
+            operation.DocumentationElement.AppendChild(newSummaryElement);
+	    }
```
Comment (BillWagner) [inline docs/framework/wcf/samples/custom-wsdl-publication.md]: Again, a stray tab character:

```suggestion
        }
```

File: docs/framework/wcf/samples/custom-wsdl-publication.md [hunk 4]
```diff
@@ -65,43 +65,46 @@ public interface ICalculator
...
+	}
```
Comment (BillWagner) [inline docs/framework/wcf/samples/custom-wsdl-publication.md]: and again:

```suggestion
    }
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this is good @Youssef1313 

I noticed a number of locations where tab characters are modifying the formatting. Can you update those?

File: docs/csharp/programming-guide/concepts/linq/how-to-query-an-arraylist-with-linq.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ ms.assetid: 2bfb471c-6e9a-4e60-bd83-4a1778abde11
 # How to: Query an ArrayList with LINQ (C#)
 When using LINQ to query non-generic <xref:System.Collections.IEnumerable> collections such as <xref:System.Collections.ArrayList>, you must explicitly declare the type of the range variable to reflect the specific type of the objects in the collection. For example, if you have an <xref:System.Collections.ArrayList> of `Student` objects, your [from clause](../../../language-reference/keywords/from-clause.md) should look like this:  
   
-```  
+```csharp
 var query = from Student s in arrList  
 ...
```
Comment (mairaw) [inline docs/csharp/programming-guide/concepts/linq/how-to-query-an-arraylist-with-linq.md]: add the comment here so it becomes "compilable"

```suggestion
//...  
```

**Comment by mairaw** [comment]
Resolved some merge conflicts after merging someone else's PR

**Comment by Youssef1313** [comment]
Fixed other merge conflicts that occurred again.

**Comment by mairaw** [comment]
There were more merge conflicts since then. Can you please check @Youssef1313?

**Comment by Youssef1313** [comment]
@mairaw, Fixed them.

File: docs/framework/wcf/feature-details/how-to-use-configuration-to-add-an-aspnet-ajax-endpoint.md [hunk 6]
```diff
@@ -35,15 +35,15 @@ Windows Communication Foundation (WCF) allows you to create a service that makes
         public double Add(double n1, double n2)  
         {  
             return n1 + n2;  
-        }  
-  
-    //Other operations omitted…  
+        }
+		//Other operations omitted…
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/how-to-use-configuration-to-add-an-aspnet-ajax-endpoint.md]: Fix indentation here.
```suggestion
        // Other operations omitted…
```

File: docs/framework/wcf/feature-details/wcf-web-http-error-handling.md [hunk 7]
```diff
@@ -9,20 +9,21 @@ Windows Communication Foundation (WCF) Web HTTP error handling enables you to re
 ## WCF Web HTTP Error Handling  
  The <xref:System.ServiceModel.Web.WebFaultException> class defines a constructor that allows you to specify an HTTP status code. This status code is then returned to the client. A generic version of the <xref:System.ServiceModel.Web.WebFaultException> class, <xref:System.ServiceModel.Web.WebFaultException%601> enables you to return a user-defined type that contains information about the error that occurred. This custom object is serialized using the format specified by the operation and returned to the client. The following example shows how to return an HTTP status code.  
   
-```  
+```csharp
 Public string Operation1()
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/wcf-web-http-error-handling.md]:
```suggestion
public string Operation1()
```

File: docs/framework/wcf/feature-details/wcf-web-http-error-handling.md [hunk 8]
```diff
@@ -9,20 +9,21 @@ Windows Communication Foundation (WCF) Web HTTP error handling enables you to re
...
+   // ...
+   throw new WebFaultException(HttpStatusCode.Forbidden);
 }  
 ```  
   
  The following example shows how to return an HTTP status code and extra information in a user-defined type. `MyErrorDetail` is a user-defined type that contains extra information about the error that occurred.  
   
-```  
+```csharp
 Public string Operation2()
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/wcf-web-http-error-handling.md]:
```suggestion
public string Operation2()
```

File: docs/framework/wcf/feature-details/wcf-web-http-error-handling.md [hunk 9]
```diff
@@ -9,20 +9,21 @@ Windows Communication Foundation (WCF) Web HTTP error handling enables you to re
...
   
  The following example shows how to return an HTTP status code and extra information in a user-defined type. `MyErrorDetail` is a user-defined type that contains extra information about the error that occurred.  
   
-```  
+```csharp
 Public string Operation2()  
    // Operation logic  
-   // ...   MyErrorDetail detail = new MyErrorDetail  
+   // ...
+   MyErrorDetail detail = new MyErrorDetail
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/wcf-web-http-error-handling.md]:
```suggestion
   MyErrorDetail detail = new MyErrorDetail()
```

**Review by Thraka — CHANGES_REQUESTED** [review]
Found a few more code problems that need to be fixed. That weird mix of VB/C# that snuck through at some point.

**Review by Thraka — APPROVED** [review]

File: docs/framework/wcf/feature-details/flowing-transactions-into-and-out-of-workflow-services.md [hunk 10]
```diff
@@ -258,52 +258,52 @@ Workflow services and clients can participate in transactions.  For a service op
...
-            }  
+        static void Main(string[] args)  
+        {  
+            //Build client  
+            Console.WriteLine("Building the client.");  
+            WorkflowApplication client = new WorkflowApplication(new DeclarativeClientWorkflow());  
+            client.Completed = Program.Completed;  
+            client.Aborted = Program.Aborted;  
+            client.OnUnhandledException = Program.OnUnhandledException;  
+
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/flowing-transactions-into-and-out-of-workflow-services.md]:
```suggestion

```

File: docs/framework/wcf/feature-details/wcf-web-http-error-handling.md [hunk 11]
```diff
@@ -9,20 +9,21 @@ Windows Communication Foundation (WCF) Web HTTP error handling enables you to re
 ## WCF Web HTTP Error Handling  
  The <xref:System.ServiceModel.Web.WebFaultException> class defines a constructor that allows you to specify an HTTP status code. This status code is then returned to the client. A generic version of the <xref:System.ServiceModel.Web.WebFaultException> class, <xref:System.ServiceModel.Web.WebFaultException%601> enables you to return a user-defined type that contains information about the error that occurred. This custom object is serialized using the format specified by the operation and returned to the client. The following example shows how to return an HTTP status code.  
   
-```  
-Public string Operation1()  
+```csharp
+public string Operation1()
 {   // Operation logic
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/wcf-web-http-error-handling.md]:
```suggestion
{
    // Operation logic  
```

File: docs/framework/wcf/feature-details/wcf-web-http-error-handling.md [hunk 12]
```diff
@@ -9,20 +9,21 @@ Windows Communication Foundation (WCF) Web HTTP error handling enables you to re
...
+   throw new WebFaultException(HttpStatusCode.Forbidden);
 }  
 ```  
   
  The following example shows how to return an HTTP status code and extra information in a user-defined type. `MyErrorDetail` is a user-defined type that contains extra information about the error that occurred.  
   
-```  
-Public string Operation2()  
+```csharp
+public string Operation2()
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/wcf-web-http-error-handling.md]:
```suggestion
public string Operation2()
{
```

File: docs/framework/wcf/samples/custom-message-filter.md [hunk 13]
```diff
@@ -26,7 +26,7 @@ class FilteringEndpointBehavior : IEndpointBehavior
   
  Two address filters are defined:  
   
-```csharp  
+```csharp
 // Matches any message whose To address contains the letter 'e'  
 class MatchEAddressFilter : MessageFilter …
```
Comment (mairaw) [inline docs/framework/wcf/samples/custom-message-filter.md]:
```suggestion
class MatchEAddressFilter : MessageFilter { }
```

File: docs/framework/wcf/samples/custom-message-filter.md [hunk 14]
```diff
@@ -26,7 +26,7 @@ class FilteringEndpointBehavior : IEndpointBehavior
   
  Two address filters are defined:  
   
-```csharp  
+```csharp
 // Matches any message whose To address contains the letter 'e'  
 class MatchEAddressFilter : MessageFilter …  
 // Matches any message whose To address does not contain the letter 'e'
```
Comment (mairaw) [inline docs/framework/wcf/samples/custom-message-filter.md]: could do the same on the line 33 as well

File: docs/framework/wcf/samples/custom-message-filter.md [hunk 15]
```diff
@@ -35,13 +35,13 @@ class MatchNoEAddressFilter : MessageFilter
   
  The `FilteringEndpointBehavior` is made configurable and allows for two different variations.  
   
-```csharp  
+```csharp
 public class FilteringEndpointBehaviorExtension : BehaviorExtensionElement
```
Comment (mairaw) [inline docs/framework/wcf/samples/custom-message-filter.md]:
```suggestion
public class FilteringEndpointBehaviorExtension : BehaviorExtensionElement { }
```

File: docs/framework/wcf/samples/saml-token-provider.md [hunk 16]
```diff
@@ -154,7 +154,7 @@ This sample demonstrates how to implement a custom client SAML token provider. A
 
      The <xref:System.IdentityModel.Selectors.SecurityTokenManager> class is used to create <xref:System.IdentityModel.Selectors.SecurityTokenProvider> for specific <xref:System.IdentityModel.Selectors.SecurityTokenRequirement> that is passed to it in `CreateSecurityTokenProvider` method. A security token manager is also used to create token authenticators and token serializer, but those are not covered by this sample. In this sample the custom security token manager inherits from the <xref:System.ServiceModel.ClientCredentialsSecurityTokenManager> class and overrides the `CreateSecurityTokenProvider` method to return the custom SAML token provider when the passed token requirements indicate that the SAML token is requested. If the client credentials class (see step 3) has not specified an assertion, the security token manager creates an appropriate instance.
 
-    ```
+    ```csharp
     public class SamlSecurityTokenManager :
```
Comment (mairaw) [inline docs/framework/wcf/samples/saml-token-provider.md]:
```suggestion
    public class SamlSecurityTokenManager : ClientCredentialsSecurityTokenManager
```

File: docs/framework/wcf/samples/saml-token-provider.md [hunk 17]
```diff
@@ -154,7 +154,7 @@ This sample demonstrates how to implement a custom client SAML token provider. A
...
      ClientCredentialsSecurityTokenManager
```
Comment (mairaw) [inline docs/framework/wcf/samples/saml-token-provider.md]:
```suggestion
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Great changes. Left some suggestions @Youssef1313!

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks for making these updates @Youssef1313 

I'm approving this PR.

@mairaw when you are happy with the changes, this is ready to :shipit:

**Comment by Youssef1313** [comment]
Fixed merge conflicts again.

**Review by mairaw — APPROVED** [review]
Thanks @Youssef1313. I've fixed some merge conflicts and we can merge this.

**Pull request merged**